lib/this_feature.rb

```diff
@@ -12,6 +12,8 @@ def self.flag(flag_name, context: nil, data: {})
   end
 
   def self.adapter_for(flag_name, context: nil, data: {})
+    return configuration.test_adapter if configuration.test_adapter
```

Comment:
I think this is always going to be set so we'd need another way to tell if we should be using the test adapter or not

Reply:
What if we just got rid of this line:
```suggestion
```

and then the configuration would have to be

```ruby
if Rails.env.test?
  config.adapters = [config.test_adapter]
end
```

It's not great but it get's us a little bit closer and we don't have to make any bigger decisions right now